apps/api_web/lib/api_web/router.ex

```diff
@@ -345,6 +352,13 @@ defmodule ApiWeb.Router do
           ##### Header
           Without an api key as a request header or in the query string, requests will be tracked by IP address and have stricter rate limit. \
           [Register for a key](/register)
+
+          The HTTP headers returned in any API response show your rate limit status:
+          | Header | Description |
+          | ------ | ----------- |
+          | `x-ratelimit-limit` | The maximum number of requests you're allowed to make per time window. |
+          | `x-ratelimit-remaining` | The number of requests remaining in the current time window. |
+          | `x-ratelimit-reset` | The time at which the current rate limit time window ends in UTC epoch seconds. |
```

Comment:
:+1: good place for this. Can we pull the text into a `@modulevar` so we don't need to duplicate it.